frontend/src/pages/LandingPage/LandingPage.tsx

```diff
@@ -1,6 +1,199 @@
+/* eslint-disable react/no-unescaped-entities */
 import React from 'react';
+import drag from './Assets/img/drag-and-drop.png';
```

Comment:
also rename this and other assets to something like `dragImg` for png assets

Reply:
y